app/views/hostgroups/_form.html.erb

```diff
@@ -23,7 +23,7 @@
 
     <div class="tab-pane active" id="primary">
       <%= select_f(f, :parent_id, parent_hostgroups, :id, :to_label, {:include_blank => true}, {:label => _('Parent'),
-                      :onchange => 'hostgroup_changed(this);', :'data-url'=> process_hostgroup_hostgroups_path,
```

Comment:
Looks like the `data-url` is needed here:
https://github.com/theforeman/foreman/blob/10c89053e3a3d933c28ff37b13a93e9e37697b90/app/assets/javascripts/host_edit.js#L332

It might be the reason https://ci.theforeman.org/job/test_develop_pr_core/1605/database=postgresql-integrations,ruby=2.5,slave=fast/testReport/junit/(root)/HostgroupJSTest/test_0004_parameters_change_after_parent_update/ is failing

Reply:
yeah, I meant to rewrite it, but it slipped my fingers :roll_eyes: :man_facepalming: